src/calendar-heatmap.js

```diff
@@ -111,31 +141,78 @@ function calendarHeatmap() {
         });
       }
 
+      if (dayNumbersInBox){
+        dayNumbers = svg.selectAll('.day-numbers')
+        .data(dateRange);  //  array of days for the last yr
+        
+        dayNumbers.enter().append('text')
+        .attr('class', 'day-numbers')
+        .attr("font-size", dayNumbersFontSz+"px")
+        .attr('x', function (d, i) {
+          var cellDate = moment(d);
+          var numSzBlank = 0;
```

Comment:
If you called it `dayTextPadding` or similar it would be a more descriptive variable name